Update extension to V1.0.0

katosdev opened the pull request:
**Update for V1.0.0**

**Changes proposed in this pull request:**
- Update extension to V1.0.0
- Add default migration values to prevent console errors with no value set in ACP

**Reviewers should focus on:**
- Installation from scratch, to ensure no console errors.
- Migrations don't wipe existing values. 

**Credits**
- @imorland for putting up with me bombarding him with questions.
- @FriendsOfFlarum/contributors for creating the extension!
- @FriendsOfFlarum/qa for assisting with any tests 

Peace out! 👍🏻 💯

davwheat commented on js/package.json:
```diff
@@ -13,7 +13,7 @@
   },
   "devDependencies": {
-    "flarum": "0.1.0-beta.16",
+    "flarum": "^1.0.0",
```
We're deprecating this package, so it can be removed.

Don't forget to `npm i` in the JS folder and commit the new lockfile :)

```suggestion
```

imorland commented:
Looks like I made some boo-boos when I updated this last, can you do me a favour and update the `js` imports in:

https://github.com/FriendsOfFlarum/cookie-consent/blob/f7631ae24b1dc4657bd68efd87b518ef0d2c9811/js/src/forum/index.js#L1
Should be `flarum/forum/app`

https://github.com/FriendsOfFlarum/cookie-consent/blob/f7631ae24b1dc4657bd68efd87b518ef0d2c9811/js/src/admin/index.js#L1
Should be `flarum/admin/app`

https://github.com/FriendsOfFlarum/cookie-consent/blob/f7631ae24b1dc4657bd68efd87b518ef0d2c9811/js/src/admin/components/CookieConsentSettingsPage.js#L1
Should be `flarum/admin/components/ExtensionPage`

katosdev commented:
> Looks like I made some boo-boos when I updated this last, can you do me a favour and update the `js` imports in:
> 
> https://github.com/FriendsOfFlarum/cookie-consent/blob/f7631ae24b1dc4657bd68efd87b518ef0d2c9811/js/src/forum/index.js#L1
> 
> 
> Should be `flarum/forum/app`
> https://github.com/FriendsOfFlarum/cookie-consent/blob/f7631ae24b1dc4657bd68efd87b518ef0d2c9811/js/src/admin/index.js#L1
> 
> 
> Should be `flarum/admin/app`
> https://github.com/FriendsOfFlarum/cookie-consent/blob/f7631ae24b1dc4657bd68efd87b518ef0d2c9811/js/src/admin/components/CookieConsentSettingsPage.js#L1
> 
> 
> Should be `flarum/admin/components/ExtensionPage`

Updated as requested.

imorland reviewed (approved):
Checks out fine locally. Well done on your first extension upgrade @katosdev :)

katosdev commented:
Package Lock updated.
